### PR TITLE
Fix signup form

### PR DESCRIPTION
**Pull request by codemonkey800**
## Description

Fixes a couple of bugs identified on the HubSpot signup form:

1. Fixes issue mentioned in #916 where button is not responsive after page transition
2. Fixes issue where user has to double click button to submit signup form

### CODE REVIEW

File: frontend/src/components/SignupForm/SignupForm.tsx [hunk 1]
```diff
@@ -128,7 +126,17 @@ export function SignupForm({ onSubmit, variant = 'default' }: Props) {
...
+            onSubmit={(event) => {
+              // TODO Fix this hack.
+              // Right now there's an issue with hubspot where the signup form
+              // shows an error after the initial submission and then passes
+              // after the 2nd one. This fixes it by doing `handleSubmit` twice
+              // to simulate user clicking on it twice. Because the form becomes
+              // invisible after submission, this code should be safe to run
+              // because `handleSubmit` will not run unless the form is visible.
+              handleSubmit(event);
+              setTimeout(() => handleSubmit(event), 150);
```
Comment (klai95) [inline frontend/src/components/SignupForm/SignupForm.tsx:138]: How is the value of `150` determined for the above line?

**Review by klai95 — COMMENTED** [review]
Will review again after the necessary code changes have been written!

File: frontend/src/components/SignupForm/SignupForm.tsx [hunk 2]
```diff
@@ -128,7 +126,17 @@ export function SignupForm({ onSubmit, variant = 'default' }: Props) {
           <p>{t('footer:signUp.success')}</p>
         ) : (
           <form
-            onSubmit={handleSubmit}
+            onSubmit={(event) => {
+              // TODO Fix this hack.
```
Comment (klai95) [inline frontend/src/components/SignupForm/SignupForm.tsx:130]: are the comments here still needed for some type of implementation?

Comment (richaagarwal) [reply frontend/src/components/SignupForm/SignupForm.tsx:130]: I am strongly in favor of leaving TODOs in there until they have been fixed!

Comment (klai95) [reply frontend/src/components/SignupForm/SignupForm.tsx:130]: Makes sense to me!

**Review by klai95 — APPROVED** [review]

**Review by manasaV3 — APPROVED** [review]

**Pull request merged**